account/account.go

```diff
@@ -54,9 +55,16 @@ func GetAccount(ctx context.Context, cfg *config.Configuration, fetcher stored_r
 
 			// unmarshal fetched account to determine if it is well-formed
 			if _, ok := err.(*json.UnmarshalTypeError); ok {
-				return nil, []error{&errortypes.MalformedAcct{
-					Message: fmt.Sprintf("The prebid-server account config for account id \"%s\" is malformed. Please reach out to the prebid server host.", accountID),
-				}}
+				// attempt to convert deprecated GDPR enforce purpose fields to resolve issue
+				completeJSON, err = ConvertGDPREnforcePurposeFields(completeJSON)
+				// unmarshal again to check if unmarshal error still exists after GDPR field conversion
+				err = json.Unmarshal(completeJSON, account)
```

Comment:
Add note here that this logic exists for backwards compatibility reasons and will eventually be removed.